chocolatey-diff/Private/Get-ChangesRegexForFile.ps1

```diff
@@ -0,0 +1,66 @@
+#Requires -version 6.0
+function Get-ChangesRegexForFile {
+    <#
+.SYNOPSIS
+    Returns a set of regular expressions for a known Chocolatey file.
+
+.DESCRIPTION
+    Returns a set of regular expressions for a known Chocolatey file.
+
+.OUTPUTS
+    System.String[] with regexes for a know Chocolatey file.
+
+.PARAMETER FileName
+    The name of the file. Only the filename with extension is expected.
+
+.EXAMPLE
+    PS > Get-ChangesRegexForFile -FileName chocolateyInstall.ps1
+    <TODO>
+#>
+    [OutputType([System.String[]],[System.String])]
+    [CmdletBinding()]
+    param (
+        [Parameter(Mandatory)][string] $FileName
+    )
+
+    $knwonExtensions = @{
```

Comment:
Well, that is an interesting way of spelling it 😆

Reply:
Fixed!